packages/publint/src/shared/constants.js

```diff
@@ -33,3 +33,63 @@ export const commonInternalPaths = [
 
 // https://github.com/npm/npm-packlist/blob/53b2a4f42b7fef0f63e8f26a3ea4692e23a58fed/lib/index.js#L284-L286
 export const licenseFiles = [/^copying/i, /^licence/i, /^license/i]
+
+// Protocol for Node.js built-in modules
+export const nodeProtocol = 'node:'
+
+// List of Node.js built-in modules
+export const builtinModules = [
```

Comment:
I don't think it would be sustainable to maintain our own list of modules here, e.g. new builtin modules, or bun/deno-specific modules. I think it should be good enough to assume paths that don't start with `.` to be some-kind of module and skip it?

Reply:
Yeah I can see it being difficult to maintain and I hadn't considered bun/deno modules either. Should I remove `isExternalDependency` as well?

Reply:
Yeah I think that can be removed too as checking the absence of the initial `.` as modules would also cover that.